assets/js/angular-components/select-subscription.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Try to use angularJS

Reply:
The js files in angular-components folder are angularJS components. I'll accept your change and I'll modify the file into an angularJS one. Please explain to me what it does.